uber/templates/macros.html

```diff
@@ -119,50 +119,51 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, collect_full_address=False, copy_address=False, name_prefix="") -%}
     {% set group_or_attendee=group if group else attendee %}
     {% set id_prefix=group.id|replace("-","") if group else attendee.id|replace("-","") %}
-    {% if not is_prereg_dealer %}
+    {% if is_dealer or collect_full_address %}
```

Comment:
That's a shame. Just to explain my thinking, the idea was "if I haven't passed the macro a variable that indicates this is the second time the macro's been called, include these scripts." Probably my issue was that I forgot to test this with collect_full_address turned off. I must have gotten all turned around.